app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -67,7 +67,7 @@ protected function validator(array $data)
             'place_of_birth' => ['required', 'string', 'max:255'],
             'date_of_birth' => ['required', 'date_format:Y-m-d'],
             'mothers_name' => ['required', 'string', 'max:255'],
-            'phone_number' => ['required', 'string', 'max:15'],
+            'phone_number' => ['required', 'string', 'min:16', 'max:18'],
```

Comment:
Im not sure if different countries have different phone number lengths.
Should we restrict that much?